driver-scala/src/it/scala/org/mongodb/scala/gridfs/GridFSObservableSpec.scala

```diff
@@ -39,7 +39,7 @@ class GridFSObservableSpec extends RequiresMongoDBISpec with FuturesSpec with Be
   private var _filesCollection: Option[MongoCollection[GridFSFile]] = None
   private var _chunksCollection: Option[MongoCollection[Document]] = None
   private val singleChunkString = "GridFS"
-  private val multiChunkString = singleChunkString.formatted(s"%${1024 * 255 * 5}s")
+  private val multiChunkString = f"${singleChunkString}%1305600s"
```

Comment:
This is bonkers - a change in scala behaviour across minor releases!

The purpose of the code is to pad the string. Depending on the **point version** of scala `singleChunkString.formatted(s"%${1024 * 255 * 5}s")` may or may not pad the string. This caused a local test to fail but I never understood why, when it passed on evergreen.

I finally found the cause when I changed all the point versions in the build and in gradle.properties and then evergreen failed also. Updated the code to use the format string literal `f"` and explicitly set the amount of padding.